examples/finance/bs_ivbp.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>In this example -we- are solving for a call option.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Done